gammapy/modeling/models/cube.py

```diff
@@ -595,7 +595,7 @@ class FoVBackgroundModel(ModelBase):
 
     tag = ["FoVBackgroundModel", "fov-bkg"]
 
-    def __init__(self, spectral_model=None, dataset_name=None):
+    def __init__(self, spatial_model=None, spectral_model=None, dataset_name=None):
```

Comment:
This change is not backwards compatible as it changes the order of the function arguments. So it should go in the last position instead. However it would be good to make the order consistent with https://docs.gammapy.org/1.0/api/gammapy.modeling.models.SkyModel.html as well. @registerrier do we have a scheme to deprecate argument order?

Reply:
I can move `spatial_model` at the end and we decide later if we want the same order as the `SkyModel` and how to deal with the deprecation in that case.